src/components/NFTCollection/Collection.tsx

```diff
@@ -1,5 +1,3 @@
-import React from 'react';
-
 import CollectionItem from './CollectionItem';
 import Card from '../Card/Card';
 import './Collection.css';
```

Comment:
Please don't use props: any, you need to set property types

Reply:
Instead of any `const Collections = (props: any) => {` you can use the following construction:
```suggestion
interface CollectionsProps {
   items: {title?: string; amount?: unknown; url?: string;}[]
}

const Collections = (props: CollectionsProps) => {
```

Reply:
Great, thank you. Is there a reason you made it optional rather than the following: 

`interface CollectionsProps {
  items: {title: string; amount: number; url: string;}[]
}

const Collections = (props: CollectionsProps) => {
  return (`

Reply:
`items: {title: string; amount: number; url: string;}[]` looks better 😉